tests/load/load_test.go

```diff
@@ -28,25 +52,61 @@ func TestE2E(t *testing.T) {
 }
 
 var _ = ginkgo.Describe("[Load Simulator]", ginkgo.Ordered, func() {
+	require := require.New(ginkgo.GinkgoT())
+
+	var env *e2e.TestEnvironment
+
+	ginkgo.BeforeAll(func() {
+		genesisPath := filepath.Join(repoRootPath, "tests/load/genesis/genesis.json")
+
+		// The load tests are allergic to high levels of evm logging, so leave it at
+		// the default level instead of raising it to debug (as the warp testing does).
```

Comment:
Could we improve this comment eg:
```suggestion
		// The load tests are flaky at high levels of logging, so leave it at
		// the default level instead of raising it to debug (as the warp testing does).
```